### PR TITLE
310 add info msg

### LINKED ISSUE #310: Info Message about an event
We need a new `msg` object that responds to the "INFO" codeword. 

This should provide information about the event in question. 

revised version
```
Event Name: <title> (<codeword>)
Duration: <date+time> to <date+time>
Description: <description>

Available: 1
Partially available: 3
Unavailable: 8
Unknown: 2
```

----------------------------------------------------
initial version
```
Event Name: <title> (<codeword>)
Duration: <date+time> to <date+time>
Description: <description>

Scheduled People (1): 
* Anthony Brown

Available People (3):
* Bob Smith
* Mary Adams
* Thomas Quine

Partially Available People (2):
* Carol Line
* Michael Smith
```

If Scheduled, Available, or Partially Available have no associated people it should just say "NONE"

### PR DESCRIPTION
**Pull request by masukomi**
added info Msg and refactored msg related things

peopleified event methods

* renamed: partial_respondents -> partially_responding_people
* renamed: respondents -> responding_people
* renamed: scheduled_people->scheduled_timecards (because that's what it was)
* added: scheduled_people (because i needed it)
* added: partially_available_people
* added: available_people
* added: unavailable_people

updated events controller
* was creating instance variables and loading even related items but never
  using them

updated _scheduled_for_event.html.erb to use new event method name

resolves #310

### CODE REVIEW

**Comment by kgf** [comment]
You should probably slow down on the scheduled people portion. People are scheduled by getting an assignment. Schedules as a model are going away.

**Comment by kgf** [comment]
When you ask for info on an event, it would be good to see what you have responded as far availability for that event. Also, seeing the four "magic" numbers too. (Available, partially available, unavailable, unknown).

**Comment by masukomi** [comment]
please note the way the new functionality in the `TemplatedBase` and how `Info` uses it.

**Comment by masukomi** [comment]
the info message in this PR looks like this: 
```
Event Name: <%=event.title%> (<%= event.id_code %>)
Duration: <%= event.start_time %> to <%= event.end_time %>
Description: <%= event.description %>

Available: <%= avp.count %>
Partially Available: <%= pavp.count %>
Unavailable: <%= uavp.count %>
Unknown: <%= lame.count %>
```

File: app/models/event.rb [hunk 1]
```diff
@@ -49,19 +53,27 @@ def partial_availabilities
...
+  def partial_responding_people
     self.partial_responses.map { |a| a.person }
   end
 
   def availabilities
     responses.available
   end
 
+  def available_people
+    responses.includes(:person).available.map{|a|a.person}.uniq
```
Comment (kgf) [inline app/models/event.rb:69]: Should line 45 look more like this one ?

**Pull request merged**